WordPress/src/main/java/org/wordpress/android/ui/ActionableEmptyView.kt

```diff
@@ -99,17 +112,38 @@ class ActionableEmptyView : LinearLayout {
         val params: RelativeLayout.LayoutParams
 
         if (isSearching) {
-            params = RelativeLayout.LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.WRAP_CONTENT)
-            layout.setPadding(0, context.resources.getDimensionPixelSize(R.dimen.margin_extra_extra_large), 0, 0)
+            params = RelativeLayout.LayoutParams(
+                    LayoutParams.MATCH_PARENT,
+                    LayoutParams.WRAP_CONTENT
+            )
+            layout.setPadding(
+                    0,
+                    context.resources.getDimensionPixelSize(R.dimen.margin_extra_extra_large),
+                    0,
+                    0
+            )
 
             image.visibility = View.GONE
             button.visibility = View.GONE
         } else {
-            params = RelativeLayout.LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.MATCH_PARENT)
+            params = RelativeLayout.LayoutParams(
+                    LayoutParams.MATCH_PARENT,
+                    LayoutParams.MATCH_PARENT
+            )
             layout.setPadding(0, 0, 0, 0)
         }
 
         params.topMargin = topMargin
         layout.layoutParams = params
     }
+
+    fun announceEmptyStateForAccessibility() {
+        val subTitle = if (!TextUtils.isEmpty(subtitle.contentDescription)) {
+            subtitle.contentDescription
+        } else {
+            subtitle.text
+        }
+
+        announceForAccessibility(String.format("%s... %s", title.text, subTitle))
```

Comment:
Using `”.”` as the separator seems good enough. And it gives enough pause. Also, Kotlin string templates FTW 🙂 

```suggestion
        announceForAccessibility("${title.text}. $subTitle")
```

Reply:
Oh yes, I  love kotlin string templates. This code was in Java so when I moved it over I forgot to convert to it :)

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/10741/commits/afe4198d20435891d4be3337d93db926e53d7591